### DISCUSSION TITLE
Running automation on electron app build -source folder vs package build

### DISCUSSION THREAD

**Discussion by ellatom** [webdriverio/webdriverio#8304] [Technical Discussion]
Hi,
When an automation is running on electron app, Should there be a difference running on source folder build(squirrel build) vs running on package build? And should anything be updated in configuration file?

Thanks.

**Comment by christian-bromann** [comment]
@goosewobbler any insights?

**Reply by goosewobbler** [reply]
@ellatom Not sure what you mean by this question, maybe give more details of your setup? 

I don't have any explicit experience with squirrel, but everything should work as long as you're specifying the path to the app binary correctly; this value is provided via appName (name of your app in `electron-builder` config) and appPath (build output dir of `electron-builder`) if you are running an `electron-builder` based workflow or binaryPath for all other cases.

Note: if you are using `electron-builder` with a squirrel target it seems that is deprecated and NSIS is now recommended on Windows.

https://www.electron.build/configuration/squirrel-windows.html
